utils/dotcms/utilities.js

```diff
@@ -48,6 +48,7 @@ class CustomError extends Error {
 
 module.exports = {
     isPage,
+    isAPIRequest,
```

Comment:
why this change?

Reply:
the app breaks if not exported (it is being used on other file)